vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -50,7 +50,10 @@ class Graph(VizroBaseModel):
 
     # Convenience wrapper/syntactic sugar.
     def __call__(self, **kwargs):
-        kwargs.setdefault("data_frame", data_manager._get_component_data(str(self.id)))
+        # This default value is not actually used anywhere at the moment since __call__ is always used with data_frame
+        # specified. It's here to match Table and AgGrid and because we might want to use __call__ more in future.
+        # If the functionality of process_callable_data_frame moves to CapturedCallable then this would move there too.
+        kwargs.setdefault("data_frame", data_manager[self["data_frame"]].load())
```

Comment:
This bit of code probably won't last very long. After #439 is merged then I don't think any code does `__call__` without specifying a `data_frame`. Removing or at least rewriting this bit of code will then fix the problem @petar-qb pointed out in https://github.com/mckinsey/vizro/pull/398#discussion_r1559120849.